src/oaklib/datamodels/vocabulary.py

```diff
@@ -146,6 +146,9 @@
 OWL_VERSION_INFO = "owl:versionInfo"
 OWL_VERSION_IRI = "owl:versionIRI"
 
+OBO_TERM = "Term"
```

Comment:
these shouldn't go in vocabulary as it's entirely an obo syntax thing - aren't these already in simple obo parser?